t/t5403-post-checkout-hook.sh

```diff
@@ -74,6 +79,25 @@ test_expect_success 'post-checkout receives the right args when not switching br
 	test $old = $new && test $flag = 0
 '
 
+test_expect_success 'post-checkout is triggered on rebase' '
+	GIT_DIR=clone2/.git git checkout -b rebase-test master &&
+	rm -f clone2/.git/post-checkout.args &&
+	GIT_DIR=clone2/.git git rebase rebase-on-me &&
+	old=$(awk "{print \$1}" clone2/.git/post-checkout.args) &&
+	new=$(awk "{print \$2}" clone2/.git/post-checkout.args) &&
+	flag=$(awk "{print \$3}" clone2/.git/post-checkout.args) &&
```

Comment:
Whoa. Three processes. Wouldn't `read old new flag <clone2.git/post-checkout.args` do the job without a spawned process?

Reply:
Done :)